Demo/Demo/DemoSettings.swift

```diff
@@ -0,0 +1,21 @@
+// swiftlint:disable convenience_type
+
+import Foundation
+
+class DemoSettings {
+
```

Comment:
To address some of Minh's feedback about making using the environment - can we make `DemoSettings` a singleton + add an `environment` property to it ? That'll let the `DemoMerchantAPI` just read from the current settings, rather than assume `sandbox`. Guessing we're already thinking in this direction, just saving it for a future PR?

Reply:
Yeh - we were planning on adding this logic into the follow up PR. Cool if we punt on it for now?